submissions/rocket-silo/rocket-silo.md

```diff
@@ -0,0 +1,21 @@
+## The Rocket Silo: Mysteries in Full <author>garr890354839</author>
+
+Ah, the rocket silo. It is the largest in-game entity outside of the crashed spaceship. It is also one of the most expensive craftable entities. However, when its placed it has wierd effects on the electric grid, stranger still if modules are at play. We also mostly know that using productivity modules inside of it pays off. What about around the silo? All of that in this week's Alt-F4!
+
+### Power Usage
+
+To start, lets look at its power usage.It claims it uses at most 4MW (which is true), but it only uses 0.25MW when just manufacturing rocket parts, or a bit more than 3 assembling machine 1s (0.225MW). When does it require 4MW, or a bit more than 51 assembling machine 1s? When the silo doors are moving, or a rocket is being pushed up. Those things must be heavy! It also uses an additional 10 kW idly when its nighttime, but there are presently no surface lights to indicate this. Back in 0.16, however, there were lights on the silo. Stranger still, when modules are applied they increase the energy usage of the silo by more than expected. 4 speed modules inside a silo should make the total energy usage of rocket part making to about 700kW, according to the mathematics behind it. But when observed, its closer to 960kW, or 2 beacons' worth of power. It does not mess with the aforementioned 4MW times like this, only this part. Speaking of modules and beacons...
```

Comment:
I think it would be best to split this into two paragraphs. Talking about beaconed power usage feels like it could maybe be expanded a bit, possibly suggest some ideas for causes of the issue? I'd also suggest expanding the bit about the lights with info from: https://forums.factorio.com/viewtopic.php?f=30&t=100109 and maybe include a picture of the silo with lights for anyone who's curious.

Reply:
The forum post has a dev reply that they are now removing the light power draw in the next release.